packages/getting-started/src/browser/style/index.css

```
@@ -89,3 +89,17 @@ body {
   text-transform: capitalize;
   font-weight: 400;
 }
+
+.gs-preference-container {
+  display: flex;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  justify-content: center;
+}
+.gs-preference {
```

minor: missing newline.

```suggestion
}

.gs-preference {
```